internal/gitutil/git.go

```diff
@@ -78,6 +79,26 @@ func GetRemoteURL(dir string) (string, error) {
 func Exec(pwd string, args ...string) (string, error) {
 	klog.V(4).Infof("Going to run git %s", strings.Join(args, " "))
 	cmd := osexec.Command("git", args...)
+
+	// NOTE: https://github.com/kubernetes-sigs/krew/pull/739
```

Comment:
let's add a bit of more explanation here, e.g.

// NOTE: on windows, msys2 and cygwin interpret args e.g. `@{upstream}` and therefore
// acts differently than git-for-windows. see https://github.com/kubernetes-sigs/krew/pull/739

Reply:
Thank you.